src/Rule/ConditionalRule.php

```diff
@@ -16,7 +16,7 @@ final class ConditionalRule implements Rule
     private $rule;
 
     /**
-     * @var callable(Password|string): bool Condition function.
+     * @var callable(Password|string):bool Condition function.
```

Comment:
My type parser currently can't deal with the space in `@var` property annotations (but can elsewhere). I removed anyway because it's neater.

Reply:
This should be part of a coding standard. The most common style for initializing the variable would be something like `function ($password): bool`, so `callable (Password|string): bool` would be the most consistent annotation. But avoiding spaces makes it easier to parse :) Do you know any coding standards that provide guidelines on this? I couldn't find it in https://github.com/consistence/coding-standard or https://github.com/slevomat/coding-standard.

Reply:
No current consensus, because this is a new annotation (I suggested it here: https://github.com/vimeo/psalm/issues/580)

Reply:
Actually this is really just a bug on my end, going to revert the change

Reply:
Fixed the bug (it allowed `) : ` but not `): `) in https://github.com/vimeo/psalm/commit/79544eb289f8fcda57e18b6c5f25c034fb121059